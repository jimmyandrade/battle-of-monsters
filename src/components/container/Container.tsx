import type { FC, HTMLAttributes } from 'react'
import { ContainerRoot } from './Container.styles'

export const Container: FC<HTMLAttributes<HTMLDivElement>> = (
  {
    children,
    ...props
  }
) => (
  <ContainerRoot {...props}>
    {children}
  </ContainerRoot>
)
