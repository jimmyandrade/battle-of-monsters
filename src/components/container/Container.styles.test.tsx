import { render } from '@testing-library/react'
import { ContainerRoot } from './Container.styles'

describe('Container styled', () => {
  describe('Root component', () => {
    it('should match snapshot', () => {
      const { container } = render(<ContainerRoot />)
      expect(container).toMatchSnapshot()
    })
  })
})
