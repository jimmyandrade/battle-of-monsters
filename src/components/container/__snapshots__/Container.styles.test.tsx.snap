// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container styled Root component should match snapshot 1`] = `
.c0 {
  margin: 0 auto;
  width: 814px;
}

<div>
  <div
    class="c0"
  />
</div>
`;
